src/adapters/postgres/user-adapter.ts

```
@@ -2271,4 +2269,68 @@ export class PostgresUserService implements IServicelocator {
   }
   
 
+  async suggestUsername(request: Request, response: Response, suggestUserDto: SuggestUserDto) {
```

ESLint

[error] Replace `request:·Request,·response:·Response,·suggestUserDto:·SuggestUserDto` with `⏎····request:·Request,⏎····response:·Response,⏎····suggestUserDto:·SuggestUserDto⏎··`

(prettier/prettier)